tests/config/indent_ctor_init.cfg

```diff
@@ -1,1289 +1,18 @@
-# Uncrustify 0.58
```

Comment:
No, that is actually quiet nice. I am slowly trying to reduce the config/test bloat as well. I wrote a somewhat faulty script for such things [here](https://gist.github.com/CDanU/54446524c8919b1d490147f6c8165bb6#file-option_reducer-py). I'll run that on the config and see if I get the same results.